docs/processes/github_access.md

```diff
@@ -42,6 +42,17 @@ You should contact the organisation owners to create a new repository. They'll n
 
 **Do not create repositories under your own user account!** Though repositories can be transferred at a later date, it is easier for everyone if they originate within our organisations.
 
+## Analysing a repository with SonarQube Cloud
+
+If your repository is in the Defra GitHub organisation, we have a SonarQube Cloud organisation that can be used to perform static analysis on your code.
+
+You should put this in place, to ensure that you comply with [our standard for code quality checks](../standards/common_coding_standards.md/#all-code-is-checked-for-quality).
+
+To request this, you can contact the organisation owners:
+
+- use the #sonar-support channel on the Defra Digital Slack workspace
```

Comment:
Should we add a comment that you need to have signed into SonarQube Cloud at least once before you can be added to the Defra organisation?